great_expectations/cli/batch_request.py

```diff
@@ -3,6 +3,12 @@
 import uuid
 from typing import Any, Dict, List, Optional, Tuple, Union, cast
 
+try:
+    from typing import Final
+except ImportError:
+    # Fallback for python < 3.8
+    from typing-extensions import Final
+
```

Comment:
I believe the import is `typing_extensions`!

Reply:
oops :)